homeassistant/components/utility_meter/sensor.py

```diff
@@ -754,6 +757,8 @@ def extra_state_attributes(self):
         # in extra state attributes.
         if last_reset := self._last_reset:
             state_attr[ATTR_LAST_RESET] = last_reset.isoformat()
+        if self._next_reset is not None:
+            state_attr[ATTR_NEXT_RESET] = self._next_reset.isoformat()
```

Comment:
It looks like this is going to change a often and fill the database up with extra attribute rows.

Should it be added to `_unrecorded_attributes`?